frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts

```diff
@@ -384,7 +384,6 @@ export const sessionRecordingDataLogic = kea<sessionRecordingDataLogicType>([
                                 ],
                                 orderBy: ['timestamp ASC'],
                                 limit: 1000000,
-                                personId: String(person.id),
```

Comment:
I don't think we can...

We do two queries

One for events with the session id
And another for the person whose session it was, in the time range of the session, but without the session id.

If we remove the person id then in the second case we query for _all_ events for that team in that time range, no?

Reply:
Oh yeah just realised that...